text/0000-change-doc-default-urls.md

```diff
@@ -0,0 +1,97 @@
+- Feature Name: N/A
+- Start Date: 2016-12-22
+- RFC PR: 
+- Rust Issue: 
+
+# Summary
+[summary]: #summary
+
+Change doc.rust-lang.org to redirect to the latest release instead of an alias
+of stable.
+
+# Motivation
+[motivation]: #motivation
+
+Today, if you hit https://doc.rust-lang.org/, you'll see the same thing as if
+you hit https://doc.rust-lang.org/stable/. It does not redirect, but instead
+displays the same documentation. This is suboptimal for multiple reasons:
+
+* One of the oldest bugs open in Rust, from September 2013 (a four digit issue
+  number!), is about the lack of `rel=canonical`, which means search results
+  are being duplicated between `/` and `/stable`, at least ([issue link][9461])
+* `/` not having any version info is a similar bug, stated in a different way,
+  but still has the same problems. ([issue link][14466])
+* We've attempted to change the URL structure of Rustdoc in the past, but it's
+  caused many issues, which will be elaborated below. ([issue link][34271])
+
+[9461]: http://github.com/rust-lang/rust/issues/9461
+[14466]: https://github.com/rust-lang/rust/issues/14466
+[34271]: https://github.com/rust-lang/rust/issues/34271
+
+There's other issues that stem from this as well that haven't been filed as
+issues. Two notable examples are:
+
+* When we release the new book, links are going to break. This has multiple
+  ways of being addressed, and so isn't a strong motivation, but fixing this
+  issue would help out a lot.
+* In order to keep links working, we modified rustdoc [to add redirects from
+  the older format](https://github.com/rust-lang/rust/issues/35020). But this
+  can lead to degenerate situations in certain crates. `libc`, one of the most
+  important crates in Rust, and included in the official docs, [had their docs
+  break](https://github.com/rust-lang/libc/pull/438) because so many extra
+  files were generated that GitHub Pages refused to serve them any more.
```

Comment:
It still affects how the crate hosts its docs, even though they're not included in the distribution.